app/scripts/modules/google/serverGroup/configure/wizard/loadBalancers/elSevenOptions/backendServiceSelector.component.js

```diff
@@ -12,7 +12,9 @@ module.exports = angular.module('spinnaker.deck.gce.backendServiceSelector.compo
     controller: function($scope) {
 
       $scope.$on('$destroy', () => {
-        delete this.command.backendServices[this.loadBalancerName];
+        if (this.command.backendServices) {
+          delete this.command.backendServices[this.loadBalancerName];
+        }
```

Comment:
@danielpeach this was throwing an exception before this PR if you added a load balancer, then removed it without ever adding a backend service